Relativity.Kepler.Client.SDK.md

```diff
@@ -0,0 +1,17 @@
+# Relativity Kepler Client SDK Version Compatibility
+
+[![nuget](https://img.shields.io/nuget/v/Relativity.Kepler.Client.SDK.svg)](https://www.nuget.org/packages/Relativity.Kepler.Client.SDK)
```

Comment:
Hi, I will approve this PR but once the package is published please make sure that the nuget link here points to the actual package in Nuget.org